### ISSUE TITLE
Feat: when rendering @types docs, combine the @types README with the library's own README

### ISSUE THREAD

**Issue by cspotcode** [TypeStrong/typedoc-auto-docs#14]
The `@types` README is informative but not useful if you're trying to actually use the library.  The library's own README is what you want, and we can pull it from npm's API.

As an example, when we render docs from @types/ahoy.js, we should still include the README from ahoy.js
https://www.npmjs.com/package/@types/ahoy.js
https://www.npmjs.com/package/ahoy.js

**Comment by cspotcode** [comment]
Hmm, maybe the API won't give us the README.  But we can grab it from unpkg.com or `npm pack`

**Comment by keithlayne** [comment]
There's probably more that we need from the actual package (not that I can really think of it right now), but I agree that's the README that we want to use.

On a similar note, what about the actual jsdocs?  For DT packages, is the jsdoc from the JS sources typically copied by hand into the DT package, or what?

With both packages on hand, we could conceivably merge info from both to generate docs.

**Comment by cspotcode** [comment]
Yeah, I think it's usually copied by hand, written by hand, or copied from elsewhere if the source package doesn't have JSDoc.
